src/TrackPanel.h

```diff
@@ -53,9 +53,7 @@ struct TrackPanelDrawingContext;
 
 enum class UndoPush : unsigned char;
 
-enum {
-   kTimerInterval = 50, // milliseconds
-};
+static constexpr auto  kTimerInterval = std::chrono::milliseconds{50};
```

Comment:
Feels like it won't harm to put duration_cast to ensure that values passed to wx* or other functions that accept not typed units work correctly even in case if someone decides to change units in the definition, now it's not stated that I'm obliged to use same units.